pkg/util/machines/machines.go

```diff
@@ -33,12 +33,12 @@ func IsMachineHealthy(c client.Client, machine *mapiv1.Machine) bool {
 
 	readyCond := conditions.GetNodeCondition(node, v1.NodeReady)
 	if readyCond == nil {
-		glog.Infof("node %s does have 'Ready' condition", machine.Name)
+		glog.V(4).Infof("node %s does have 'Ready' condition", machine.Name)
```

Comment:
this does not seems related to the title of this commit

Reply:
done